Alignment/CommonAlignmentProducer/python/ALCARECOTkAlMuonIsolatedHI_cff.py

```diff
@@ -0,0 +1,33 @@
+# AlCaReco for track based alignment using isolated muon tracks
+import FWCore.ParameterSet.Config as cms
+
+from Alignment.CommonAlignmentProducer.ALCARECOTkAlMuonIsolated_cff import *
+
+ALCARECOTkAlMuonIsolatedHIHLT = ALCARECOTkAlMuonIsolatedHLT.clone(
+    eventSetupPathsKey = 'TkAlMuonIsolated'
+    )
```

Comment:
@jrcastle I am wondering, why di you add separate `eventSetupPathsKey` for all the new producers (`TkAlJpsiMuMuHI`, `TkAlUpsilonMuMuHI`, `TkAlZMuMuHI`) but not for this one? The tests are failing because the keys you are requiring are new and are not availble in GT. If you can align them to the pp ones it should run fine. But there might be specific reasons for you to keep them separate. The pp ones are currently empty (meaning that they don't select any particular trigger path) this might not be the case you wish for HI. Please let me know, and we can act accordingly.